users/vieting/experiments/librispeech/librispeech_100_ctc/fairseq_finetuning/ctc_standalone/pytorch_networks/wav2vec/w2v_ctc_wrapper_cfg.py

```diff
@@ -0,0 +1,80 @@
+from dataclasses import dataclass
+import dataclasses
+
+from fairseq.models.wav2vec.wav2vec2_asr import Wav2Vec2CtcConfig
+from fairseq.dataclass.configs import FairseqConfig
+from fairseq.tasks.audio_finetuning import AudioFinetuningTask, AudioFinetuningConfig
+from fairseq.tasks import FairseqTask
+from fairseq.data import Dictionary
+
+from omegaconf import OmegaConf
+
+@dataclass
+class ModelConfig:
+    """
+    Model configuration for Wav2Vec2CTC Wrapper Model.
+
+    Args:
+        model_config_updates (dict): Updates to the Wav2Vec2CtcConfig
+            (see fairseq.models.wav2vec.wav2vec2_asr.Wav2Vec2CtcConfig)
+        task_config_updates: dict: Updates to the AudioFinetuningConfig
+            (see fairseq.tasks.audio_finetuning.AudioFinetuningConfig)
+        label_target_size: int: Size of the target dictionary.
+    """
+    model_config_updates: dict = dataclasses.field(default_factory=dict)
+    task_config_updates: dict = dataclasses.field(default_factory=dict)
+    label_target_size: int = 0
+
+    def build_full_config(self) -> FairseqConfig:
+        """
+        Build a full FairseqConfig object from the model and task configurations updates.
+        Non-specified fields will be filled with the default values.
+
+        FairseqConfig is needed to instantiate a FairseqModel.
+        """
+        fairseq_model_config = self._build_model_config()
+        fairseq_task_config = self._build_task_config()
+        fairseq_conf = FairseqConfig()
+        setattr(fairseq_conf, 'model', fairseq_model_config)
+        setattr(fairseq_conf, 'task', fairseq_task_config)
```

Comment:
Can this be written as
```suggestion
        fairseq_conf = FairseqConfig(
            model=fairseq_model_config,
            task=fairseq_task_config,
        )
```